### PR TITLE
Implement function to mark tasks as done

### LINKED ISSUE #19: Implement function to mark tasks as done
Design a done function such that users can mark their task as 'done' in their task list.

### PR DESCRIPTION
**Pull request by yAOwzers**

### CODE REVIEW

**Review by franceneee — DISMISSED** [review]
Looks okay, but looking at the checks, I don't think the code works now o.o Something probably went wrong after the merge conflict was resolved?

File: src/main/java/seedu/duke/userinterface/command/Done.java [hunk 1]
```diff
@@ -1,8 +1,36 @@
...
+import seedu.duke.userinterface.AppState;
+import seedu.duke.userinterface.CliMessages;
+import seedu.duke.userinterface.CliUserInterface;
+
 public class Done extends CliCommand {
+
+    public static final String COMMAND_WORD = "done";
+
+    private AppState appState;
+    private String arguement;
```
Comment (franceneee) [inline src/main/java/seedu/duke/userinterface/command/Done.java]: argument* , same for the others

**Comment by NeilBaner** [comment]
Looks like there's probably some missing imports for exceptions.

**Review by NeilBaner — CHANGES_REQUESTED** [review]
Not passing the CI and not building correctly. I suspect some error with missing imports for exceptions.

File: src/main/java/seedu/duke/userinterface/command/Done.java (hunk 1 above)
Comment (yAOwzers) [reply src/main/java/seedu/duke/userinterface/command/Done.java]: Oh okay thanks! Will make edits! I think something went wrong during the merge conflict, was working fine on my end just now

**Review by franceneee — COMMENTED** [review]
Ooops, thanks @NeilBaner for correcting me. xD @yAOwzers do remember to run checkStyle as well!

**Review by franceneee — APPROVED** [review]

**Pull request merged**